content/guides/references/cypress-studio.md

```diff
@@ -1,14 +1,28 @@
 ---
 title: Cypress Studio
+e2eSpecific: true
```

Comment:
Above screenshot seems usually wide?

Also tip - if you hold SPACE during the screenshot, you can get the real clean "capture window only" style like the previous ones - good to keep in mind.